lib/representatives.js

```diff
@@ -245,10 +245,24 @@ import { Err } from "./helpers/errors.js";
    * @param {string} representativeID - Identifier of representative to retrieve
    * @param {RepresentativeCreateUpdate} representative - Representative to add
    * @returns {Promise<Representative>}
+   * @deprecated PUT call to representatives is deprecated, please use PATCH.
```

Comment:
Being that the call was `update` let's just change the underlining call from PUT to PATCH.  The user experience will be the same between the two.  And we would not need to update the documentation for that then.

Reply:
Won't the call fail? The user will continue to send a PUT request and there is nothing to tell them that PATCH is the new one.

Reply:
If the change was going from a PATCH to a PUT, then it could fail.  But since a PUT (as to how it originally was for the update method) requires the full object, it will not fail.  The user is just making a method call to update.  They do not know that underneath it is using a PUT or PATCH API call.